AndroidSDKCore/src/main/java/com/leanplum/LeanplumInbox.java

```diff
@@ -85,6 +85,13 @@ public static void disableImagePrefetching() {
     isInboxImagePrefetchingEnabled = false;
   }
 
+  /**
+   * Enable prefetching images.
+   */
+  public static void enableInboxImagePrefetching() {
```

Comment:
You shouldn't change original class when you write tests. Please remove this method.

Reply:
I disagree, While writing the tests I discovered that calling disableImagePrefetching can not be undone, It makes a lot of sense for this method to be there and should have been created in the first place.